tests/test_acceptance.py

```diff
@@ -173,9 +209,9 @@ class Meta:
             float_field = FloatField(name="floatField", required=True, allow_null=False)
             boolean_field = BooleanField(name="boolField", required=True, allow_null=False)
             datetime_field = DateTimeField(name="datetimeField", required=True, allow_null=False)
-            date_field = DateField(name="dateField", required=True, allow_null=False)
-            time_field = TimeField(name="timeField", required=True, allow_null=False)
-            uuid_field = UuidField(name="uuidField", required=True, allow_null=False)
+            date_field = DateField(name="dateField", required=True, allow_null=True)
```

Comment:
This stems from the fact that the model being used here has invalid values. For me to fix this I would need to do either change the model to not use null values for fields where allow_null=False or revert changes made in the Serializer object that cause the model to be validated. If the model is not validated it is possible to get invalid values for fields when they are not supplied by the object being serialized. This scenario was possible before this PR when using the Serializer.data_to_object() method. The Model was being used but not validated and then returned.